lib/handlers/check_channel_connection_handler.dart

```diff
@@ -0,0 +1,93 @@
+import 'dart:async';
+
+import 'package:breez/bloc/account/account_bloc.dart';
+import 'package:breez/logger.dart';
+import 'package:breez/theme_data.dart' as theme;
+import 'package:breez/widgets/flushbar.dart';
+import 'package:flushbar/flushbar.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:rxdart/rxdart.dart';
+
+class CheckChannelConnection {
+  static final _instance = CheckChannelConnection._internal();
+  final _subscriptions = CompositeSubscription();
+  final _notReadySubscriptions = CompositeSubscription();
+
+  Flushbar _flushbar;
+
+  factory CheckChannelConnection() => _instance;
+
+  CheckChannelConnection._internal();
+
+  void startListen(
```

Comment:
How about something like the following code (didn't test it) to simplify the implementation?

```
  Timer notReadyTimer;
  
  accountBloc.accountStream.
  map((acc) => acc.readyForPayments).
  distinct().
  listen((ready) {
    if (ready) {
      // remove flushbar
      notReadyTimer?.cancel();
    } else {
      notReadyTimer = Timer(Duration(seconds: 30), (){
        // show not ready flushbar
      });
    }
  });
```

Reply:
🤔  how can I cancel these tasks?

Reply:
What do you mean by tasks?

Reply:
The "remove/show flushbar", on my tests this cancel and start again improved a lot the flushbar behaviour